hw/ip/entropy_src/doc/_index.md

```diff
@@ -104,70 +166,45 @@ After power-up, the ENTROPY_SRC block is disabled.
-Clearing the bit in this register will prevent future modification of the {{< regref "ES_CONF" >}} register or the  {{< regref "ES_SEED" >}} by firmware.
+For simplicity of initialization, only a single register write is needed to start functional operation of the ENTROPY_SRC block.
+This assumes that proper defaults are chosen for thresholds, sampling rate, and other registers.
 
-The {{< regref "ES_SEED" >}} register can be updated at any time while the digital source FIFO is in operational mode.
-As long as the update sequence is followed by firmware, there will be no timing hazards when accessing these registers by the firmware.
+For security reasons, a configuration and control register locking function is performed by the {{< regref "REGEN" >}} register.
+Clearing the bit in this register will prevent future modification of the {{< regref "CONF" >}} register or other writeable registers by firmware.
 
 ### Entropy Processing
 
 When enabled, the ENTROPY_SRC block will generate entropy bits continuously.
-The `es_entropy_valid` bit signal to the firmware when entropy bits should read from the {{< regref "ES_ENTROPY" >}} register.
-While the firmware is reading the TLUL FIFO, additional entropy bits could be captured by the TLUL FIFO as long as the TLUL FIFO is not full.
-The firmware will do 32-bit register reads of the {{< regref "ES_ENTROPY" >}} register to retrieve the entropy bits.
-Each read will automatically pop an entry off the head of the TLUL FIFO.
-New entropy bits can enter the TLUL FIFO at the tail concurrently.
-
-The hardware entropy interface will move entropy bits out of the HWIF FIFO when it is not empty and the downstream hardware is ready.
-The HWIF FIFO has lower priority than the TLUL FIFO, but is expected to have higher bandwidth demands.
-Because of this, higher priority is given to the TLUL FIFO so that it does not starve when the hardware entropy interface is consuming a high level of entropy bits.
-
-The {{< regref "ES_THRESH" >}} register is used to set the point of notification to the firmware.
-Notification can be either by polling or by an interrupt.
-For debug, the {{< regref "ES_FDEPTHST" >}} register can be read to find out the current state of the TLUL FIFO at any time.
-
-An additional feature is the {{< regref "ES_RATE" >}} register.
-The purpose of this register is to simulate faster or slower entropy generation sources when is digial source mode.
-This will aid firmware design and debug when trying to handle all entropy source types and rates.
-When in rng mode, this register must exactly match the sample rate required by the rng source, expected to be in the range of about 25 to 50 kilohertz.
+The `es_entropy_valid` bit will indicate to the firmware when entropy bits can read from the {{< regref "ENTROPY_DATA" >}} register.
+The firmware will do 32-bit register reads of the {{< regref "ENTROPY_DATA" >}} register to retrieve the entropy bits.
```

Comment:
Suggest: "The firmware reads 32 bits of entropy at a time from the ENTROPY_DATA register."